mods/screwdriver/init.lua

```diff
@@ -19,6 +11,72 @@ screwdriver.rotate_simple = function(pos, node, user, mode, new_param2)
 	end
 end
 
+-- For attached wallmounted nodes: returns true if rotation is valid
+local function check_attached_node(pos, rotation)
```

Comment:
sure, but it'll suffice for now. I'll insert a reference though, to make it clear what it is.